spec/system/candidate_interface/carry_over/candidate_carries_over_unsubmitted_application_with_a_course_to_new_cycle_spec.rb

```diff
@@ -78,9 +73,7 @@ def and_the_cancel_unsubmitted_applications_worker_runs
   end
 
   def when_i_sign_in_again
-    logout
-    login_as(@candidate)
-    visit root_path
+    given_i_am_signed_in_with_one_login
```

Comment:
```suggestion
    click_link_or_button 'Sign out'
    given_i_am_signed_in_with_one_login
```

I think we still want to logout if we want to retain the original meaning of the test.